### PR TITLE
bugfix: add platform value to container image list 

### PR DESCRIPTION
**Pull request by kakaZhou719**
### Describe what this PR does / why we need it
1. add platform to container image list;
2. skip download duplicate container image;
3. add some debug logs

### CODE REVIEW

File: pkg/image/save/save.go [hunk 1]
```diff
@@ -71,14 +71,22 @@ func (is *DefaultImageSaver) SaveImages(images []string, dir string, platform v1
...
+	existFlag := make(map[string]struct{})
 	//handle image name
 	for _, image := range images {
 		named, err := ParseNormalizedNamed(image, "")
 		if err != nil {
 			return fmt.Errorf("failed to parse image name:: %v", err)
 		}
 
-		//check if image exist
+		//check if image is duplicate
```
Comment (starnop) [inline pkg/image/save/save.go:82]: This judgment may be redundant, as the OCI pull process itself checks for the presence of the image layer.

In addition, there is a risk of misjudgment here.

Comment (kakaZhou719) [reply pkg/image/save/save.go:82]: 1. i wonder there is not redundant.
for example: input images["nginx:latest","docker.io/library/nginx:latest"], will be transformed ["docker.io/library/nginx:latest","docker.io/library/nginx:latest"], so we need to do deduplication，nothing to do with pull process logic.

2. what risks here ? sorry i dont understand?

Comment (starnop) [reply pkg/image/save/save.go:82]: Sorry, I thought we were doing the duplicate removal in the step of pulling image. Currently, it seems that only for the image name, which is OK

**Review by starnop — APPROVED** [review]
LGTM

**Pull request merged**